packages/blade/src/components/Badges/_decisions/decisions.md

```diff
@@ -0,0 +1,61 @@
+# Badge <!-- omit in toc -->
+Badges are used to show small amount of color coded metadata, which are ideal for getting users attention.
+
+<img src="./badge-thumbnail.png" width="380" />
+
+## Design
+- [Figma - Badge](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8110%3A417)
+
+
+## API
+
+| Prop | Type | Required | Default | Description |
+|---|---|---|---|---|
+| variant | `positive`,`negative`, `notice`, `information`, `neutral` | No | `neutral` | The variant of the badge to be rendered. |
+| contrast | `low`, `high` | No | `low` | The contrast of the badge to be rendered. |
+| size | `medium`, `small` | No | `medium` | The size of the badge to be rendered. |
+| children | `string` | **Yes** | undefined | The label to be rendered within the badge. |
+| icon | `Icon` | No | undefined | The Blade `Icon` component to be rendered for the badge. |
+| labelStyle | `italic`, `normal` | No | `normal` | The font style of the badge's label. |
+| labelWeight | `bold`, `regular` | No | `regular` | The font weight of the badge's label. |
+
+### Sample Usage
+```jsx
+import { Badge } from '@razorpay/components';
+
+<Badge 
+  variant='positive'
+  contrast='high'
+  icon={InfoIcon} 
+  size='large' 
+  labelStyle='italic' 
+  labelWeight='bold'
+>
+  New
+</Badge>
+```
+
+## Open Questions
+#### Dev
+- Should the label be automatically capitalized when `labelWeight` is `bold`
+- Should we call it `labelWeight`, `labelStyle` or `fontWeight`, `fontStyle` or `textWeight`, `textStyle`?
+- Should we call it `variant` or `intent`?
+- What should be the default `variant`? Kept it as `neutral` for now since it seems more _neutral_ and appropriate
+- Scope of A11y?
```

Comment:
@anuraghazra I don't think we explicitly need to do anything here. Do you think we might need to do something extra?

Reply:
On further digging. Should we make the aria-role as `status`. It acts as a `live-region`

Ref: 
- https://www.w3.org/TR/wai-aria-1.1/#status
- https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22#example-2-updating-the-shopping-cart-status
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role

Reply:
I think if we're considering icon only badges then a label would definitely be needed

Reply:
Yes icon only badges need that. 

Tho @chaitanyadeorukhkar we don't need role=status in a badge. Mostly badges are static elements. And that Add to cart example is very specific which can be achieved with our announce() function. 

Unfortunately there's not pattern for badges thus better not to invent one.

Reply:
We won't be having icon only badge so this won't be a concern